config/sync/block.block.breadcrumbs.yml

```diff
@@ -17,9 +17,4 @@ settings:
   label: Breadcrumbs
   provider: system
   label_display: '0'
-visibility:
-  request_path:
-    id: request_path
-    pages: "/news\r\n/publications\r\n"
-    negate: true
-    context_mapping: {  }
+visibility: {  }
```

Comment:
Do you need the breadcrumbs exposed as a block? They usually available in the preprocess_page hook to pass into the template for rendering as a Twig variable. A block might be handy if you need to swap the regions for it but otherwise I think this might be overkill for what you need.